### ISSUE TITLE
test & ppa builds fail

### ISSUE THREAD

**Issue by timmie** [statsmodels/statsmodels#2200]
```

statsmodels.tsa.tests.test_tsa_tools.test_add_lag_drop_noinsert ... ok
statsmodels.tsa.tests.test_tsa_tools.test_freq_to_period ... ok
SKIP: X13/X12 not available

======================================================================
ERROR: Failure: TypeError (<functools.partial object at 0x2ae5e2e8caf8> is not a function or method)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/failure.py", line 42, in runTest
    raise self.exc_class(self.exc_val)
TypeError: <functools.partial object at 0x2ae5e2e8caf8> is not a function or method

======================================================================
ERROR: Failure: TypeError (<functools.partial object at 0x2ae5e2e8c6d8> is not a function or method)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/failure.py", line 42, in runTest
    raise self.exc_class(self.exc_val)
TypeError: <functools.partial object at 0x2ae5e2e8c6d8> is not a function or method

======================================================================
ERROR: Failure: TypeError (<functools.partial object at 0x2ae5e33f5890> is not a function or method)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/failure.py", line 42, in runTest
    raise self.exc_class(self.exc_val)
TypeError: <functools.partial object at 0x2ae5e33f5890> is not a function or method

```

**Comment by josef-pkt** [comment]
Thanks @timmie  and thanks again for setting up the unit testing.

I looked at your tests results earlier today. Skipper has a fix that will be merged today #2199.
It's a problem with older nose versions, but we fix those problems so we don't need to increase the required nose version.
Also Debian had this problem #2121

**Comment by josef-pkt** [comment]
duplicate of #2121
fix was merged in #2199
python-xy Ubuntu testing is all green
